docs/development-environment-cli-support.md

```diff
@@ -16,7 +16,7 @@ You need GIT for contributing to Jetpack.
 
 **n** is a node version manager. Similar and alternative to **nvm**.
 
-### Command is available: php
+### Command is available: PHP
```

Comment:
Since we're talking about a command, should this remain lowercase? Is there a way to mark a line for exclusion from spellchecking? (I'm thinking of the case or two in where we *want* `WordPress` spelled `wordpress` because it is an array key or something.)

Reply:
Sure, that can remain lowercase. The `.spelling` file allows per-file whitelisting.